### PR TITLE
Add ability to provide other solutions in PywrRandomGenerator.

### LINKED ISSUE #887: Injected Population to Platypus
Hi @jetuk 

I was looking at the `PywrRandomGenerator` class to see a way to inject a complete set of non-dominated solutions into `Platypus` to continue an algorithm search from the last non-dominated solutions I got from a previous search. However, I noticed that using this `pywr` class I can inject only one non-dominated solution (the solution there is in the Json file). I would like to ask you two questions:

1) Do you know if it is ok just inject one solution to continue the search? or I would need to inject a complete set of non-dominated solutions equal to the pop-size?
2) If you recommend me to inject the complete set of non-dominated solutions. Could you share some ideas about how this could be efficiently implemented in `pywr`? I could dedicate time to do this implementation but it would be helpful to get some tips from you.

Thanks

https://github.com/pywr/pywr/blob/e7b4e35c5eef08442e1f390c4a5b8270cb0ae926/pywr/optimisation/platypus.py#L135

### PR DESCRIPTION
**Pull request by jetuk**
This extends the ability of PywrRandomGenerator to return alternative initial solutions instead of just the current model configuration. User's provide a list of dictionaries containing variable values for each of the variable Parameters.

Fixes #887.

### CODE REVIEW

File: pywr/optimisation/platypus.py [hunk 1]
```diff
@@ -135,33 +135,62 @@ def evaluate(self, solution):
 class PywrRandomGenerator(platypus.RandomGenerator):
     """A Platypus Generator that injects the current setup of the Pywr model into the population.
```
Comment (Batch21) [inline pywr/optimisation/platypus.py:136]: Need to update this line as you now use the class without injecting the current setup

File: tests/test_platypus.py [hunk 2]
```diff
@@ -80,11 +80,38 @@ def test_platypus_nsgaii_process_pool(two_reservoir_problem):
...
+    def test_other_solutions(self, two_reservoir_problem, use_current):
+        """Test PywrRandomGenerator inserts other solutions into the population. """
+
+        # Create some alternative initial solutions
+        solutions = [
+            {
+                'control_curve': {'doubles': [1] * 12}
+            },
+            {
+                'control_curve': {'doubles': [2] * 12}
```
Comment (Batch21) [inline tests/test_platypus.py:103]: Might be worth extending this so it tests setting integer variables

Comment (jetuk) [reply tests/test_platypus.py:103]: I just looked at doing this. We currently have no parameters with `integer_size > 0`. So we'll have to park this.

File: pywr/optimisation/platypus.py (hunk 1 above)
Comment (jetuk) [reply pywr/optimisation/platypus.py]: Done.

**Pull request merged**